app/questionnaire/value_source_resolver.py

```diff
@@ -34,8 +34,8 @@ class ValueSourceResolver:
     schema: QuestionnaireSchema
     location: LocationType | None
     list_item_id: str | None
-    progress_store: ProgressStore
-    supplementary_data_store: SupplementaryDataStore
+    progress_store: progress_store.ProgressStore
```

Comment:
for consistency are we able to  import progress_store as ps like in `placeholder_renderer.py`